capabilities/display_capabilities.js

```diff
@@ -847,82 +767,7 @@ SDL.templateCapabilities = {
             },
             "numCustomPresetsAvailable": 8
         },
-        "buttonCapabilities": [{
-            "name": "PRESET_0",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "PRESET_1",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "PRESET_2",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "PRESET_3",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "PRESET_4",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "PRESET_5",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "PRESET_6",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "PRESET_7",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "PRESET_8",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "PRESET_9",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "OK",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "SEEKLEFT",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "SEEKRIGHT",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "TUNEUP",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "TUNEDOWN",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }],
+        "buttonCapabilities": SDL.ButtonCapability.filter(button => button.name !== 'CUSTOM_BUTTON'),
```

Comment:
```suggestion
        "buttonCapabilities": SDL.ButtonCapability,
```
Seems to me there is no need to omit this value, since it is supported in this template

Reply:
@jacobkeeler Ommited on [develop branch](https://github.com/smartdevicelink/sdl_hmi/blob/d58988ec3e1d00f619bcb88d68f5e3aff2fec312/capabilities/display_capabilities.js#L850)
I tried to leave it as a development branch.
Are you sure that is shouldn't be ommited?

Reply:
I believe it should be included, yes. Regardless, this should be made consistent between templates. Currently only the `NAV_FULLSCREEN_MAP` includes this button, but it's supported by all templates.

Reply:
@jacobkeeler Implemented 5563ddf80f22964bb8696a1735d8f2604accec06